platforms/software/antlr/src/test/groovy/org/gradle/api/plugins/antlr/AntlrPluginTest.groovy

```diff
@@ -110,4 +110,13 @@ class AntlrPluginTest extends AbstractProjectBuilderSpec {
         def sourcesJar = project.tasks.sourcesJar
         sourcesJar.taskDependencies.getDependencies(null).contains(generateGrammarSource)
     }
+
+    @Issue('https://github.com/gradle/gradle/issues/33373')
```

Comment:
We generally only use `@Issue` for regression tests. 

We should remove this line

Reply:
Applied the feedback : )